ui/update_status.go

```diff
@@ -85,6 +85,12 @@ func (ui *UserInterface) updateStatus(sleepTime time.Duration) {
 			ui.positionTotal = 0
 		}
 
+		// Update volume status
+		if castVolume != nil {
+			ui.volume = int(castVolume.Level * 100)
```

Comment:
Good catch, it looks like these aren't being set anywhere. Can this be moved down to the top of the other `if castVolume != nil {` block?

Reply:
Sure, I only did it this way to match the way that the position updates were done.